run.py

```diff
@@ -1221,6 +1258,15 @@ def custom_openapi():
         action="store_true",
         help="指定すると音声合成を途中でキャンセルできるようになります。",
     )
+    parser.add_argument(
+        "--enable_guided", action="store_true", help="入力音声を解析してAudio Queryで返す機能を有効化します。"
```

Comment:
```suggestion
        "--enable_guided", action="store_true", help="入力音声を解析して音声合成クエリで返す機能を有効化します。"
```
https://github.com/VOICEVOX/voicevox_engine/blob/92f00d704a6f6ebcccfa31e35e5a97758b56a1da/voicevox_engine/synthesis_engine/synthesis_engine.py#L398-L399

synthesis_engine.pyでは「音声合成クエリ」でした